src/concepts/sql_style.md

```diff
@@ -59,6 +63,45 @@ LIMIT
   10
 ```
 
+### Joins
+
+Always include the `JOIN` type rather than relying on the default join.
+
+**Good**
+```sql
+SELECT
+  submission_date,
+  e.key AS experiment_id,
+  e.value AS experiment_branch,
+  count(*) AS count
+FROM
+  telemetry.clients_daily
+CROSS JOIN
+  UNNEST(experiments) AS e(key, value)
+WHERE
+  submission_date > '20190701'
+  AND sample_id = '10'
+GROUP BY
+  1, 2, 3
```

Comment:
*ahem* *ahem* _explicit_ *ahem* *ahem* ;)

Reply:
I *knew* this would come up :)

In Presto SQL, you cannot refer to the aliased column name in your `GROUP BY`, you must repeat the column expression (which can get ugly if it's more complicated than this example). In that case, I prefer the `GROUP BY 1` syntax over `GROUP BY experiment.key`. Particularly when it's `GROUP BY date_trunc(date_parse(...time string...), DAY)`

I just found out, though, that BigQuery *does* let you `GROUP BY` the aliased column name! Hooray!

Reply:
That is great news!! I propose we make that the default in group-by expressions for BQ.